rpcs3/Emu/Cell/Modules/cellNetCtl.cpp

```diff
@@ -125,20 +125,25 @@ error_code cellNetCtlGetInfo(s32 code, vm::ptr<CellNetCtlInfo> info)
 {
 	cellNetCtl.todo("cellNetCtlGetInfo(code=0x%x (%s), info=*0x%x)", code, InfoCodeToName(code), info);
 
+	if (code == CELL_NET_CTL_INFO_ETHER_ADDR)
+	{
+		// dummy mac address
+		std::memset(info->ether_addr.data, 0xFF, sizeof(info->ether_addr.data));
+		return CELL_OK;
+	}
+
+	if (g_cfg.net.net_status == CELL_NET_CTL_STATE_Disconnected)
+	{
```

Comment:
Elseif

Reply:
If it enters the IF clause it will return and exit the function, elseif is completely unnecessary, unless it's some kind of compiler optimization

Reply:
Aha yep, my mistake